plugins/atlas/atlas.go

```diff
@@ -0,0 +1,25 @@
+package atlas
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func MongoDBAtlasCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "MongoDB Atlas CLI", // TODO: Check if this is correct
+		Runs:      []string{"atlas"},
+		DocsURL:   sdk.URL("https://www.mongodb.com/docs/atlas/cli/v1.2/"),
```

Comment:
nit: how often do we expect atlas cli to release new versions? Given this link contains a specific version, we could consider using this link: https://www.mongodb.com/docs/atlas/getting-started/ which shows how to get started including directions to the how to install page